relayer/chains/cosmos/cosmos_chain_processor.go

```diff
@@ -402,11 +404,11 @@ func (ccp *CosmosChainProcessor) queryCycle(ctx context.Context, persistence *qu
 		})
 
 		if err := eg.Wait(); err != nil {
-			ccp.log.Warn(
-				"Could not query block data. Consider checking if your RPC node is online, and that transaction indexing is enabled.",
```

Comment:
we already have this log here. https://github.com/cosmos/relayer/blob/5236f6f5635c639b02d8949d30882d2f9ab31b64/relayer/chains/cosmos/tx.go#L481

this log fires almost constantly because there is a delay between when an rpc node reports a newer height and when that height is available to be queried. It is confusing to the operator because it makes it seem like there is a transaction indexing problem on the RPC node but it's really just typical operation.